pkg/splunk/enterprise/clustermaster_test.go

```diff
@@ -261,23 +269,33 @@ func TestPerformCmBundlePush(t *testing.T) {
 		Data: map[string]string{configToken: ""},
 	}
 
+	_, err = splctrl.ApplyConfigMap(client, &smartstoreConfigMap)
```

Comment:
Check to err here again? If err in cm creation

Reply:
Addressed it.